ingest/src/projectSources.ts

```diff
@@ -504,6 +506,53 @@ const practicalAggregationsDataSource = async () => {
   return await makeMdOnGithubDataSource(practicalAggregationsConfig);
 };
 
+export const terraformProviderSourceConstructor = async () => {
+  const siteBaseUrl =
+    "https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs";
+  return await makeGitDataSource({
+    name: "atlas-terraform-provider",
+    repoUri: "https://github.com/mongodb/terraform-provider-mongodbatlas.git",
+    repoOptions: {
+      "--depth": 1,
+      "--branch": "master",
+    },
+    metadata: {
+      productName: "mongodbatlas Terraform Provider",
+      tags: ["docs", "terraform", "atlas", "hcl"],
+    },
+    filter: (path: string) =>
+      path.includes("website/docs") && path.endsWith(".markdown"),
+    handlePage: async function (path, content) {
+      const { metadata, body } = extractFrontMatter<{ page_title: string }>(
```

Comment:
page_title: have we used this case style before? Any particular reason for introducing it here?

Reply:
this comes from the frontmatter in the markdown file we're ingesting. not something we're defining